setup.py

```diff
@@ -0,0 +1,13 @@
+from setuptools import setup
+
+setup(
+    name='stylize-datasets',
+    version='1.0',
```

Comment:
I'd prefer version 0.1.0, this package is far from stable/release ready ;)

Reply:
i would actually suggest 0.1, and release on pypi TBH

Reply:
@MMathisLab 0.1.0 is the pypi way to write 0.1 ;) as the changes in this PR are required for pypi, I'll upload it once this is merged

Reply:
yes, sorry, I meant re: "this package is far from stable/release ready ;)" I would say it's better to make a release is what I meant (and yes, of course as you like 0.1.0 is of course great too ;)